src/nxdoc/tutorials/transversal-how-tos/how-to-implement-local-groups-or-roles-using-computed-groups.md

```diff
@@ -146,6 +146,21 @@ In this how to, you will learn how to let managers of a workspace determine who
 
 {{/callout}}
 
+Starting from 8.10 HF-04 you need to add the following contribution.
+
+{{#> panel type='code'}}
+
+```xml
+<extension point="configuration" target="org.nuxeo.runtime.ConfigurationService">
```

Comment:
I guess we need to require the default properties to be loaded first, to ensure the property can be overwritten properly:

```xml
<extension point="configuration" target="org.nuxeo.runtime.ConfigurationService">
    <require>org.nuxeo.ecm.core.automation.core.properties</require>
    ...
</extension
```